e2e-app/setup.pw-spec.ts

```diff
@@ -11,6 +11,23 @@ beforeAll(async() => {
   } catch (e) {
     console.error('Unable to setup a new page with playwright', e);
   }
+
+  test.page.on('console', async msg => {
+    let hesUnexpectedMessage = true;
+    const consoleMsgs: Array<string> = ['Unexpected console messages found'];
+    for (let i = 0; i < msg.args().length; ++i) {
+      const message: string = await msg.args()[i].jsonValue();
+      if (message.includes('Angular is running') || message.includes('Live Reloading')) {
```

Comment:
We should simply turn off live reloading for tests, like it is done for the demo for example:
- run demo with a specific config for playwright `yarn e2e-app:serve -c playwright`
- similar to https://github.com/ng-bootstrap/ng-bootstrap/blob/master/angular.json#L155-L158

I guess it should be something like:

```
"playwright": {
  "browserTarget": "demo:build:production",
  "liveReload": false,
  "watch": false,
  "extractLicenses": false
}
```